pkg/component/observability/plutono/dashboards/seed/fluent-bit-dashboard.json

```
@@ -1200,51 +1799,48 @@
                 "value": 80
               }
             ]
-          }
+          },
+          "unit": "ms"
         },
         "overrides": []
       },
       "gridPos": {
-        "h": 6,
-        "w": 12,
-        "x": 12,
-        "y": 27
+        "h": 8,
+        "w": 13,
+        "x": 11,
+        "y": 45
       },
-      "hideTimeOverride": true,
-      "id": 54,
+      "id": 101,
       "options": {
-        "frameIndex": 0,
-        "showHeader": true
+        "graph": {},
+        "legend": {
+          "calcs": [
+            "last"
+          ],
+          "displayMode": "table",
+          "placement": "right"
+        },
+        "tooltipOptions": {
+          "mode": "single"
+        }
       },
-      "pluginVersion": "7.2.1",
+      "pluginVersion": "7.5.42",
       "targets": [
         {
-          "expr": "sum(increase(fluentbit_vali_gardener_errors_total{pod=~\"$pod\"}[$__rate_interval]) != 0) by (host)",
-          "format": "time_series",
-          "instant": false,
+          "exemplar": true,
+          "expr": "sum(rate(valitail_request_duration_seconds_sum{pod=~\"$pod\",host=~\".+$host.+\"}[5m])) by (host)\n/\nsum(rate(valitail_request_duration_seconds_count{pod=~\"$pod\",host=~\".+$host.+\"}[5m])) by (host)\n* 1000",
```

As far as I know, expressions in Plutono should use `[$__rate_interval]` instead of a hard-coded intervals like `[5m]`. 

@vicwicker , @rickardsjp correct me if I am wrong.

Unfortunately, this is not documented as a guideline in the [Adding Plutono Dashboards](https://github.com/gardener/gardener/blob/d7bb834ac93f5eb18cebf2e7ae9e86653b5e826f/docs/development/monitoring-stack.md#L150). FYI @vicwicker , @rickardsjp.

Suggestion:
```suggestion
          "expr": "sum(rate(valitail_request_duration_seconds_sum{pod=~\"$pod\",host=~\".+$host.+\"}[$__rate_interval])) by (host)\n/\nsum(rate(valitail_request_duration_seconds_count{pod=~\"$pod\",host=~\".+$host.+\"}[$__rate_interval])) by (host)\n* 1000",
```